starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,37 @@
-// Names and Input
+let hacker1 = "Alfredo";
+console.log(`The driver's name is ${hacker1}`);
```

Comment:
Big fan of the newer string concatenation syntax using backticks. Very good habit to get into early!